README.md

```diff
@@ -51,7 +51,37 @@ Global:
  * _WebUI/State_
    * `fortigate_last_reboot_seconds`
    * `fortigate_last_snapshot_seconds`
-
+ * _SwitchController/ManageSwitch/PortStats_  
+   * `fortiswitch_status`
+   * `fortiswitch_port_status`
+   * `fortiswitch_port_speed_bps`
+   * `fortiswitch_port_transmit_packets_total`
+   * `fortiswitch_port_transmit_bytes_total`
+   * `fortiswitch_port_transmit_unicast_packets_total`
+   * `fortiswitch_port_transmit_multicast_packets_total`
+   * `fortiswitch_port_transmit_broadcast_packets_total`
+   * `fortiswitch_port_transmit_errors_total`
+   * `fortiswitch_port_transmit_drops_total`
+   * `fortiswitch_port_transmit_oversized_packets_total`
+   * `fortiswitch_port_receive_packets_total`
+   * `fortiswitch_port_receive_bytes_total`
+   * `fortiswitch_port_receive_unicast_packets_total`
+   * `fortiswitch_port_receive_multicast_packets_total`
+   * `fortiswitch_port_receive_broadcast_packets_total`
+   * `fortiswitch_port_receive_errors_total`
+   * `fortiswitch_port_receive_drops_total`
+   * `fortiswitch_port_receive_oversized_packets_total`   
+ * _SwitchController/ManageSwitch/Health_
+   * `fortiswitch_health_summary_cpu`
```

Comment:
* Avoid using words like mem when memory works
 * These health metrics names have no unit in them (e.g. add _celsius to`fortiswitch_health_temperature`